CVCs, CDs, and CCs (#108)
makerdao/endgame-docs: maker-core/delegates.md

# Constitutional Delegates (CDs)

## Overview

> Delegates are ecosystem actors who operate through Protocol Delegation System smart contracts. MKR holders can delegate their voting power to them without the need to hand over the custody of their MKR.

Reviewer: ```suggestion
Constitutional Delegates (CDs) are [delegates](https://manual.makerdao.com/delegation/overview/what-is-delegation) who are also [Constitutional Conservers](https://endgame.makerdao.com/maker-core/tbc-governance/constitutional-conservers). MKR holders can delegate their MKR to Constitutional Delegates through the [Easy Governance Frontend](easy-governance-frontend.md)

Unlike ordinary delegates, CDs are eligible for compensation and are subject to certain restrictions. CDs must execute governance strategies defined by [Constitutional Voter Committees](cvc.md).
```

> Constitutional Delegates (CDs) are one type of [Constitutional Conservers](https://endgame.makerdao.com/maker-core/tbc-governance/constitutional-conservers). They receive various benefits and are subject to specific requirements. They must put in place the governance strategies defined by the CVCs.

Reviewer: Would delete if you accept the above suggestion

> CDs must not engage with other ecosystem actors or in operational activities. They are not allowed to compensate MKR holders that delegate to them. They must aim to provide governance information and research material to MKR holders.

Reviewer: They must aim to provide governance information and research material to MKR holders - is this not done by CVCs? Or is it both?

Author: I will change it for "CVCs."

From MIP101 latest amendments:

##### 6.5: Constitutional Delegate Alignment Risk Mitigation

CDs are generally not allowed to engage directly with Ecosystem Actors and other counterparties to the DAO, or in other ways engage in operational activities. They must remain focused on publicly providing governance information and research material to CVCs, and can interact with ecosystem participants for this sole purpose

## Purpose

> 

Reviewer: ```suggestion
```

The Constitutional Delegate role spares Delegates from having to create governance strategies. CDs must follow strategies that have been designed by CVC members.

> CDs do not take part in the creation of the strategies, they execute them. The risk of Delegates following their own agenda, since they are not required to hold any MKR, is then  eliminated.

Reviewer: Not sure if this comes under "Purpose". I'd say something along the lines of "CDs exist in order to allow MKR holders to delegate their MKR and also support a CVC strategy." or something

## Income

> CDs can receive compensation from the Maker Protocol. The compensation goes to the top 14 CDs that have the highest amount of combined MKR delegated to their delegate contracts and fulfill the participation requirements.

Reviewer: ```suggestion
CDs can receive compensation from the Maker Protocol. The compensation goes to the top 14 CDs that have the highest amount of total MKR delegated to their delegate contracts and fulfill the CD eligibility requirements.
```

There are two income levels for the top 14 qualified CDs: Prime and Reserve.  The 7 top qualified CDs are called the Prime Constitutional Delegates (PCDs). The top 8 to 14 qualified CDs are called the Reserve Constitutional Delegates (RCDs).

> Each income level has a monthly total compensation.

Reviewer: A total monthly compensation of xxx MKR/DAI is distributed equally among eligble Prime CDs. Similarly, a total monthly compensation of yyy MKR/DAI is distributed equally among eligible Reserve CDs.

Author: I thought we agreed on leaving specific numbers out since they are subject to changes?

Author: @LongForWisdom, any thoughts?

Qualified CDs can earn an equal share of their income level total compensation. If a CD loses its qualification, their slot goes to the next qualified CD by the amount of delegated MKR.

### Eligibility Requirements
To get a share of the total compensation of their income level, CDs must fulfill the following requirements:
1) Participation in CVC Subcommittee Meetings (monthly)
    - PCDs must participate in 16+

>     - RDCs must participate in 4+

Reviewer: ```suggestion
    - RCDs must participate in 4+
```

2) Voting Activity (yearly)
    - CDs must be active in 95%+ of all votes
3) Communication Activity (yearly)
    - CDs must communicate reasoning for their voting in 95%+ of all votes

> If these amounts are not reached, then CDs receive a reduced amount of the total CD income they are eligible for.

Author: Thank you! I somehow omitted this part: "For Voting and Communication activity, it reaches zero at 75%. If it falls below 75%, they are no longer qualified for a CD income and the next eligible CD takes the slot."

## Visibility

The EGF displays all CDs who follow two different CVC Governance Strategies. To do this, the CD must have two separate Protocol Delegation System smart contracts. Each of them is attached to the CVC whose Governance Strategy they follow. To calculate the income level, the total amounts of MKR delegated to both Governance Strategies of the CD are counted.

> When a [CVC](https://endgame.makerdao.com/maker-core/tbc-governance/cvc) bans a CD, the MKR holders decide through a vote if the CD can continue to be displayed with one Independent Governance Strategy.

Reviewer: ```suggestion
When a CVC bans a CD, the MKR holders decide through a vote if the CD can continue to be displayed with one Independent Governance Strategy.
```

Reviewer: Also, "the MKR holders" - does this refer to only those who delegated to that CD or all MKR holders?

Author: It has been updated on MIP101. It is through a CVC decision.
"If a CD is banned by a CVC decision, the votes delegated for the CD to the strategy will instead count as a part of the delegates independent strategy."
